### PR TITLE
Add `rel="noopener"` attribute for external links

### PR DESCRIPTION
**Pull request by rebeccawong7**
Re-submission of this closed PR: #347 

In this PR the `noopener` attribute has been added for external links. The `rel` and `target` attributes have not been moved to the end of line.

Contributes toward optimization issue: #331 

Potentially unblocks: #351

### CODE REVIEW

File: src/views/components/package-header.html [hunk 1]
```diff
@@ -21,18 +21,18 @@ <h1 class="package-name">{{name}}</h1>
 				</a>
 				{{/if}}
 				{{#if githubRepo}}
-				<a target="_blank" rel="nofollow" href="https://github.com/{{githubRepo.user}}/{{githubRepo.project}}/" class="button" title="GitHub">
+				<a target="_blank" rel="nofollow" rel="noopener" href="https://github.com/{{githubRepo.user}}/{{githubRepo.project}}/" class="button" title="GitHub">
```
Comment (MartinKolarik) [inline src/views/components/package-header.html:24]: This needs to be `rel="nofollow noopener"`

Comment (rebeccawong7) [reply src/views/components/package-header.html]: Thanks, done

**Comment by jimaek** [comment]
@MartinKolarik Any idea why tests are failing?

**Comment by MartinKolarik** [comment]
👍

**Pull request merged**